framework/Samplers/EnsembleForward.py

```diff
@@ -68,10 +69,16 @@ def localInputAndChecks(self,xmlNode):
         self.instanciatedSamplers[child.tag] = returnInstance(child.tag,self)
         #FIXME the variableGroups needs to be fixed
         self.instanciatedSamplers[child.tag].readXML(child,self.messageHandler,variableGroups={},globalAttributes=self.globalAttributes)
-      elif child.tag in knownTypes():
```

Comment:
what about the Sampler we specify in the EnsembleForward is not among the known one?